hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -288,6 +302,9 @@ private void processRecords(ManagedRocksIterator iterator,
         batch = new ArrayList<>(batchSize);
         sequenceId++;
       }
+      if ((preFileRecords > -1) && (count >= preFileRecords)) {
```

Comment:
Seems like the `ldb` will generate unlimited empty file If the `preFileRecords` is zero.